tests/pipelines/stable_diffusion/test_stable_diffusion_img2img.py

```diff
@@ -197,6 +219,36 @@ def test_save_load_optional_components(self):
     def test_attention_slicing_forward_pass(self):
         return super().test_attention_slicing_forward_pass()
 
+    @skip_mps
+    def test_pt_np_pil_outputs_equivalent(self):
```

Comment:
Super cool tests! Let's keep them here for now - we can then later add the to `test_pipeline_common.py` to apply them to all relevant pipelines.